src/ui/common/src/components/workflows/nodes/Node.tsx

```diff
@@ -65,22 +99,19 @@ export const Node: React.FC<Props> = ({
   }
 
   let backgroundColor;
-  if (execState?.status === ExecutionStatus.Succeeded) {
+  if (status === ExecutionStatus.Succeeded) {
```

Comment:
We should probably turn this into a switch statement of sorts. maybe even a function of it's own if this is something that we plan to use in other parts of our UI. (Can't think of any at the top of my head right now though 🤔 )